programs/adapter-tulip/src/lib.rs

```diff
@@ -165,37 +165,39 @@ pub mod adapter_tulip {
                 let mut token_b_account_and_balance =
                     load_token_account_and_balance(ctx.remaining_accounts, 5);
 
-                // reference: https://github.com/sol-farm/tulipv2-sdk/blob/main/vaults/src/instructions/deposit_tracking.rs#L38
-                let mut withdraw_deposit_tracking_data =
-                    sighash("global", "withdraw_deposit_tracking").try_to_vec()?;
-                withdraw_deposit_tracking_data.append(&mut input_struct.share_amount.try_to_vec()?);
-                withdraw_deposit_tracking_data.append(&mut input_struct.farm_type_0.try_to_vec()?);
-                withdraw_deposit_tracking_data.append(&mut input_struct.farm_type_1.try_to_vec()?);
+                let mut is_double_dip = false;
+                if ctx.remaining_accounts.len() == 33 {
```

Comment:
What does magic number `33` mean here?

Reply:
Well... this is how I distinguish is a orca vault with or without double dip.

Reply:
1. Noted. How about adding some descriptive comment here
2. What is the account number for non-double-dip vaults?

Reply:
1. Sure
2. non-double-dip: 33
double-dip: 31 (have less accounts is because some accounts are extracted to preIx)

Reply:
Just wonder: how to distinguish double-dip and non-double-dip is accounts are not extracted to preIx. Are they both `33`?

Reply:
Nope, double-dip needs 44 accounts if not extracted to pre-instructions, non-double-dip won't change since it didn't need to extract atm.

Reply:
Noted. Plz describe the difference as detail as possible in comment :)